addons/l10n_din5008_purchase/i18n/l10n_din5008_purchase.pot

```diff
@@ -16,61 +16,61 @@ msgstr ""
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "<span class=\"fw-bold\">Shipping Address:</span>"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Cancelled Purchase Order"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Cancelled Purchase Order No.:"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Incoterm:"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Order Date:"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Order Deadline:"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Order Reference:"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Purchase Order"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Purchase Order No.:"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Purchase Representative:"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
 msgid "Request for Quotation"
 msgstr ""
 
 #. module: l10n_din5008_purchase
-#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_purchaseorder_document
+#: model_terms:ir.ui.view,arch_db:l10n_din5008_purchase.report_common_purchase_din5008_template
```

Comment:
Wont this break the translations for everyone who didn't upgrade the module?

I think adding both comments is safer for the stable version